pkg/metrics/metrics.go

```diff
@@ -11,27 +11,26 @@ import (
 const Async = "async"
 const Sync = "sync"
 const ImagePullTimeKey = "pull_duration_seconds"
-const ImageMountTimeKey = "mount_duration_seconds"
+const ImagePullTimeHistKey = "pull_duration_seconds_hist"
 const OperationErrorsCountKey = "operation_errors_total"
 
-var ImagePullTime = prometheus.NewHistogramVec(
+var ImagePullTimeHist = prometheus.NewHistogramVec(
 	prometheus.HistogramOpts{
 		Subsystem: "warm_metal",
-		Name:      ImagePullTimeKey,
+		Name:      ImagePullTimeHistKey,
 		Help:      "The time it took to pull an image",
-		Buckets:   []float64{0, 1, 5, 10, 15, 30, 60, 120, 180},
+		Buckets:   []float64{1, 5, 10, 15, 30, 60, 120, 300, 600, 900},
 	},
-	[]string{"operation_type"},
```

Comment:
there is no sync vs async in this implementation... they actual pull is identical. this PR changes the logic surrounding the previously sync-only puller. whether sync or async, the original `remoteimage.Pull()` is used relatively unchanged.